cas-server-support-osf/src/main/java/io/cos/cas/authentication/handler/support/OpenScienceFrameworkPrincipalFromRequestRemoteUserNonInteractiveCredentialsAction.java

```diff
@@ -565,6 +565,7 @@ protected PrincipalAuthenticationResult notifyRemotePrincipalAuthenticated(
         final String fullname = user.optString("fullname").trim();
         final String givenName = user.optString("givenName").trim();
         final String familyName = user.optString("familyName").trim();
+        final String isMemberOf = user.optString("isMemberOf").trim();
```

Comment:
`optString` guarantees that an empty string is returned if the key `isMemberOf` is not found. This is why don't have to add `isMemberOf` to other institutions ...